src/SerilogWeb.Classic/Classic/SerilogWebClassicConfiguration.cs

```diff
@@ -66,6 +68,7 @@ internal SerilogWebClassicConfiguration Edit(Func<SerilogWebClassicConfiguration
         internal bool IsEnabled { get; }
 
         internal LogEventLevel RequestLoggingLevel { get; }
+        internal Func<HttpContextBase, TimeSpan, LogEventLevel> RequestContextLogLevel { get; }
```

Comment:
nit: could we try a name that makes it clearer that this is a function somehow ? 

How about 

```suggestion
        internal LogEventLevel RequestLoggingLevel { get; }
        internal Func<HttpContextBase, TimeSpan, LogEventLevel> LogLevelEvaluator { get; }
```

or something like this ?